Makefile.am

```diff
@@ -2468,6 +2468,7 @@ ad_common_tests_SOURCES = \
     src/providers/ldap/sdap_async_initgroups_ad.c \
     $(NULL)
 ad_common_tests_CFLAGS = \
+    $(AM_CFLAGS) \
```

Comment:
The similar issue is also in other tests:
refcount_tests_CFLAGS
fail_over_tests_CFLAGS
sbus_tests_CFLAGS
sbus_codegen_tests_CFLAGS

and you will need to use discard_const in sbus* test to fix warnings